pkg/util/dirs.go

```diff
@@ -104,17 +106,14 @@ func CacheDir() (string, error) {
 	return path, nil
 }
 
+// EnvironmentsDir to more easily handle multiple clusters and minimise issues around
+// using old clones of environment git repositories lets use temporary directories for environment dirs
 func EnvironmentsDir() (string, error) {
-	h, err := ConfigDir()
+	dir, err := ioutil.TempDir("", "jx-env-")
```

Comment:
And how are all these temp directories get deleted?

Reply:
good point - probably best to wait  until we've time for a more comprehensive solution to this issue...